test-requirements.txt

```diff
@@ -1,4 +0,0 @@
-types-Deprecated>=1.2.9
-coveralls>=3.3.1
```

Comment:
coverage command line missing, you need to add it to test dependencies, most likely was previously installed as dependency of coveralls

Reply:
oops. should be fixed. (until we actually use coveralls.io, I switched this to the `coverage` package, though.)